### PR TITLE
Code is now PEP8-capable

### PR DESCRIPTION
**Pull request by SynneK1337**
I made code compatible with PEP-8 Style Guide

## Description

As the title said.

## Related Issue

https://github.com/adobe/ops-cli/issues/73

## Motivation and Context

It makes the code much cleaner and easier to maintain.

## How Has This Been Tested?

I used tests attached in ```tests\``` directory within Docker Contained created by modified [image](https://github.com/adobe/ops-cli/pull/16/commits/241432becad8d1ec5fde050378355017c9236f75#diff-3254677a7917c6c01f55212f86c57fbf)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/ops/inventory/ec2inventory.py [hunk 1]
```diff
@@ -188,18 +200,19 @@ def get_host_info_dict_from_instance(self, instance):
...
             elif key == 'ec2__previous_state':
                 instance_vars['ec2_previous_state'] = instance.previous_state or ''
                 instance_vars['ec2_previous_state_code'] = instance.previous_state_code
-            elif type(value) in integer_types or type(value) == bool:
+            elif type(value) in integer_types or isinstance(value, bool):
                 instance_vars[key] = value
             elif type(value) in string_types:
                 instance_vars[key] = value.strip()
-            elif type(value) == type(None):
+            elif isinstance(value, type(None)):
```
Comment (costimuraru) [inline src/ops/inventory/ec2inventory.py]: would `elif value is None:` be more clear here? As suggested in the example from https://github.com/adobe/ops-cli/issues/73

Comment (SynneK1337) [reply src/ops/inventory/ec2inventory.py]: I changed it, but in my opinion mixing ```isinstance()``` with ```is None``` is not very clear. Unfortunately, the ```is <type>``` statement only works with ```None``` type.

**Comment by amuraru** [comment]
This looks good.
Can we also add a pep8 linter check to check at build time.
And add a note in readme that pep8 is required moving fw?

**Comment by SynneK1337** [comment]
I vote for 99 characters too, I don' t think 79 is proper for 16:9 or more wide monitors nowadays.

**Comment by costimuraru** [comment]
+1 for 99 characters

**Comment by SynneK1337** [comment]
Could anyone help me with resolving these conflicts?

**Comment by amuraru** [comment]
> Could anyone help me with resolving these conflicts?

@SynneK1337 - it should be the contributor (in this case you) who needs to resolve these.

In your `SynneK1337:master` branch, something along these lines:
```
git fetch origin #<-- this should be adobe repo
git rebase -p origin/master
git push SynneK1337 master:master
```

**Comment by costimuraru** [comment]
You can also resolve the conflicts from the interface. Just click "Resolve conflicts" here and fix them.

**Comment by amuraru** [comment]
@costimuraru @serban21 https://github.com/OrkoHunter/pep8speaks/blob/master/.github/README.md a neat app we can we use moving fw

**Comment by SynneK1337** [comment]
Thanks, I'll do this after a recover my OS Partition XD

**Comment by SynneK1337** [comment]
I'll fix those lines to match 99 chars limit soon

**Comment by amuraru** [comment]
I enabled Pep8Speaks app in our repo for auto checks on PR.
We need to also override the configuration per our agreement here: 99 lines len:

https://github.com/OrkoHunter/pep8speaks/blob/master/.github/README.md

**Comment by SynneK1337** [comment]
I left few lines longer than 99 characters because I think that making them shorter will make code harder to read.

**Comment by costimuraru** [comment]
@SynneK1337 can you try to please squash all commits?

**Comment by SynneK1337** [comment]
I've done it.

**Comment by costimuraru** [comment]
Awesome, @SynneK1337. This is starting to look quite good.
Do you mind fixing the remaining issues reported by @pep8speaks (https://github.com/adobe/ops-cli/pull/74#issuecomment-540052761)? Seems like they are pretty easy ones

**Comment by SynneK1337** [comment]
I left few longer than 99 characters lines because I think that making them shorter will make code harder to read. But I can fix those unused variables issues.

**Comment by SynneK1337** [comment]
I don' t know why flake8 on my local machine doesn' t print few of those errors. I'm running default config with only ```max-line-length``` changed.

**Comment by SynneK1337** [comment]
```
PS C:\Users\synnek\ops-cli> flake8 src/ops/cli/terraform.py --max-line-length=99
src/ops/cli/terraform.py:17:1: F401 'distutils.version.StrictVersion' imported but unused
src/ops/cli/terraform.py:19:1: F401 'pkg_resources' imported but unused
src/ops/cli/terraform.py:65:100: E501 line too long (100 > 99 characters)
src/ops/cli/terraform.py:69:100: E501 line too long (100 > 99 characters)
src/ops/cli/terraform.py:70:100: E501 line too long (105 > 99 characters)
src/ops/cli/terraform.py:85:100: E501 line too long (107 > 99 characters)
src/ops/cli/terraform.py:137:100: E501 line too long (105 > 99 characters)
src/ops/cli/terraform.py:154:91: W291 trailing whitespace
PS C:\Users\synnek\ops-cli>
```

**Pull request merged**